recipes/pyccl/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyccl" %}
+{% set version = "2.0.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/CCL/archive/v{{ version }}.tar.gz
+  sha256: aaa1ee1e258abd4ffbea1fad814992b8ebef6891e4d8e43b5556122322f370f9
+  patches:
+    - remove_setuptools_scm.patch
```

Comment:
Could you please clarify one thing? Why are you removing ``setuptools_scm`` and patching the ``cmake`` commands?

Reply:
github tarballs do not have the right information for setuptools_scm to compute the version information. The eventual plan is to build this from the pypi sdist, which should work fine and won't need that part of the patch. 

I am patching the `cmake` commands in the `setup.py` so that I can use the custom `cmake` build in the `build.sh`. This lets me force the library to use `OpenMP` on OSX (which is not enabled by default) and makes debugging build issues easier.

Reply:
Thanks for the explanation! :)